fullstack-cert/css-projects/colored-boxes-layout/styles.css

```
@@ -0,0 +1,72 @@
+h1 {
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 70%;
+  height: auto;
+  padding: 10px;
+  margin: 20px auto;
+  height: 400px;
+  /* Show campers the various align-content values */
+  /* align-content: space-between; */
+  /* align-content: start; */
+  /* align-content: center; */
+  /* align-content: end; */
+  /* align-content: space-around; */
+  align-content: space-evenly;
+}
+
+.box {
+  flex: 0 0 250px;
+  /*For flex-grow, start here and state that the first value reps the flex-grow proeprty for all the boxes (making the default 0, which they can override later)*/
+  /*To demonstrate flex-shrink, ask campers to change the second value here to 1 so they can see some boxes become smaller than the others, and that the default (which can be overriden in other boxes) is set here)*/
+  /* After going through order and flex-grow, tell campers to modify the third value here (flex-basis) to observe how flex-grow distribution changes due to different line breaks and remaining space */
+  max-height: 120px;
+  color: #fff;
+  border: 1px solid #000;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 10px;
+  font-weight: bold;
+  font-size: 18px;
```

we should avoid using pixels for font size. 

```suggestion
  font-size: 1.125rem;
```